core-test/src/main/java/org/openstack4j/api/compute/ServerTests.java

```diff
@@ -67,7 +68,7 @@ protected Service service() {
     
     @Test
     public void evacuateServer() throws Exception {
-        respondWith(JSON_SERVER_CREATE);
+        respondWith(JSON_SERVER_EVACUATE);
         
         ServerPassword password =  osv3().compute().servers().evacuate("e565cbdb-8e74-4044-ba6e-0155500b2c46", EvacuateOptions.create().host("server-test-1").onSharedStorage(false));
         assertNotNull(password.getPassword());        
```

Comment:
One last thing: Instead of `assertNotNull(..)`maybe `assertEquals(password.getPassword(), "MySecretPass")`